.travis.yml

```diff
@@ -1,12 +1,16 @@
 language: python
 python:
+  - '2.7'
+  - '3.4'
+  - '3.5'
   - '3.6'
 
 install:
-  - pip install --upgrade nox
+  - pip install tox-travis
 
 script:
-  - nox -f noxfile.py -s docs
+  # - nox -f noxfile.py -s docs
```

Comment:
Please open a ticket or leave a TODO instead of the commented line for docs here.

Reply:
Docs step is already included. Removing comment.